### PR TITLE
11 location model

### PR DESCRIPTION
**Pull request by LucasAnderson1984**
Finished adding Location model, controller, views and tests.

### CODE REVIEW

File: app/controllers/locations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  end
+
+  def new
+    render_new
+  end
+
+  def create
+    if location.save
+      redirect_to location_path(location),
+                  notice: I18n.t('location.create.success')
```
Comment (josephbridgwaterrowe) [inline app/controllers/locations_controller.rb]: @LucasAnderson1984 if you use the plural `locations` then you can take advantage of some of the i18n conventions in Rails:

From http://guides.rubyonrails.org/i18n.html

4.1.4 "Lazy" Lookup

Rails implements a convenient way to look up the locale inside views. When you have the following dictionary:

```
es:
  books:
    index:
      title: "Título"

you can look up the books.index.title value inside app/views/books/index.html.erb template like this (note the dot):

<%= t '.title' %>

Automatic translation scoping by partial is only available from the translate view helper method.
```

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :locations
```
Comment (josephbridgwaterrowe) [inline config/routes.rb]: For whatever reason I feel more comfortable with devise at the top, possibly alphabetic comfort, habit, insanity.

Thoughts?

File: db/migrate/20160211194435_create_locations.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateLocations < ActiveRecord::Migration
+  def change
+    create_table :locations do |t|
+      t.integer :location_id, null: false
+      t.integer :area_id, null: false
```
Comment (josephbridgwaterrowe) [inline db/migrate/20160211194435_create_locations.rb]: Is `area_id` a "foreign key" to something?

Comment (LucasAnderson1984) [inline db/migrate/20160211194435_create_locations.rb]: Not that I know of.  I used what they were called in Grossman, e.g. InvPallet_MainInfo.  I could leave off the _id.

File: spec/policies/location_policy_spec.rb [hunk 4]
```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+describe UserPolicy, type: :policy do
```
Comment (josephbridgwaterrowe) [inline spec/policies/location_policy_spec.rb]: `describe LocationPolicy...`

**Comment by josephbridgwaterrowe** [comment]
@LucasAnderson1984 looks good to me, time for a rebase?

**Comment by LucasAnderson1984** [comment]
Good to go.

**Comment by josephbridgwaterrowe** [comment]
@LucasAnderson1984 :+1:

**Comment by josephbridgwaterrowe** [comment]
@LucasAnderson1984 don't forget, if you included Closes #11 in your commit message the issue would have been closed for you :smile:

**Pull request merged**